### PR TITLE
feat/add-guide-footer

### PR DESCRIPTION
**Pull request by moT01**
This adds a footer to all guide articles with instructions on how to contribute to the article.

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

This doesn't exactly close the issue, check it out for more details...
related to https://github.com/freeCodeCamp/freeCodeCamp/issues/35490

### CODE REVIEW

File: client/src/components/layouts/components/guide/guideFooter.js
Comment (raisedadead) [inline client/src/components/layouts/components/guide/guideFooter.js]: We need a more elegant solution than hard coding this here? You should create this slugs dynamically. See [more details here](https://www.gatsbyjs.org/docs/node-apis/)

**Review by raisedadead — CHANGES_REQUESTED** [review]

**Comment by raisedadead** [comment]
@moT01 how is this coming along?

**Comment by moT01** [comment]
Slowly, I got sidetracked and kind of put it on the back burner, but didn't forget about it.

**Comment by moT01** [comment]
Work in progress - any push in the right direction is welcome.

**Comment by moT01** [comment]
okay, I pushed another new solution. I think I actually got it right this time. I might need to do a little clean up.

**Review by raisedadead — COMMENTED** [review]
Hi @moT01 

I just ran through the code on GitHub and seems you are there. Do you mind cleaning up the commits and squash them for me to pull down and review?

[Making them conventional](https://www.conventionalcommits.org/en/v1.0.0-beta.2/#why-use-conventional-commits) would be ideal while you are at it.

Thanks a lot for all your hard work on this.

**Comment by moT01** [comment]
✔️ ?

**Review by raisedadead — APPROVED** [review]
LGTM.

Thanks @moT01 , I have gone ahead and made a few cosmetic tweaks to the footer text. Waiting for the CI and a second pair of eyes before merging.

File: client/src/templates/Guide/components/GuideFooter.js
Comment (ValeraS) [inline client/src/templates/Guide/components/GuideFooter.js]: let's redo this to a function component

File: client/src/templates/Guide/GuideArticle.js
Comment (ValeraS) [inline client/src/templates/Guide/GuideArticle.js]: How about posting this in an article layout?

**Review by ValeraS — CHANGES_REQUESTED** [review]

File: client/src/templates/Guide/components/GuideFooter.js
Comment (moT01) [reply client/src/templates/Guide/components/GuideFooter.js]: I believe the new changes should resolve this.

File: client/src/templates/Guide/GuideArticle.js
Comment (moT01) [reply client/src/templates/Guide/GuideArticle.js]: I wasn't exactly sure what you meant here, I took it as moving the `GuideFooter` component to the `ArticleLayout` component. So I did that, let me know if it's not what you wanted.

**Comment by moT01** [comment]
Thanks for the review @ValeraS - After some struggles with git, I think I got it how you want. I left comments on your review things there.

**Review by ValeraS — APPROVED** [review]

**Review by raisedadead — APPROVED** [review]

**Pull request merged**
